rpc_jobs/standard_job_fi.yml

```diff
@@ -0,0 +1,139 @@
+- job-template:
```

Comment:
Yeah we haven't really worked out how this will trigger yet, because the PR will be against the releases repo and that PR will need to trigger figure out which component is releasing and then trigger this job.  I'm pretty sure this job will need tweaking once we get that locked down.

Reply:
OK, I guess I anticpated this job being triggered directly by the PR. Any particular reason for the intermediate job?

Reply:
@git-harry wouldn't the actual tagging happen post-merge?

Reply:
@mattt416 I would say not, if it's in the repo it's been releases. If the tag is created after the commit is merged into releases, there is a window during which the the repo lists releases that don't exist.

Reply:
@git-harry @hughsaunders ok makes sense to remove `common.build_failure_issue()` from this template -- any objection to just replace it with `common.safe_jira_comment()` similar to the premerge job so that we have _something_ reporting failure until the release job is complete?

Reply:
@git-harry I could see that workflow working if we could only trigger the write actions (tag cutting for example) to happen when the PR has been approved by 2 reviewers.  Doing it before would feel a bit odd.

Reply:
@mattt416 I guess that's the L1/L2 split. L1 maybe just needs to lint the change, L2 run all the tests, tag, merge.